### PR TITLE
Bot Actions API

### PR DESCRIPTION
**Pull request by tausifmuzaffar**
GET, POST /botActions
GET, POST, DELETE, PATCH /botActions/key

### CODE REVIEW

File: api/v1/swagger.yaml [hunk 1]
```diff
@@ -8028,6 +8028,240 @@ paths:
...
+          name: queryBody
+          in: body
+          required: true
+          schema:
+            type: object
+            description: Create botAction properties
+            properties:
+              name:
+                type: string
+                maxLength: 254
```
Comment (shriramshankar) [inline api/v1/swagger.yaml]: The name validation is missing here too. If this is something that you just started adding, we can ignore it for now and you can take care of it in the next pr. Let me know.

Comment (tausifmuzaffar) [inline api/v1/swagger.yaml]: Ill do another PR to clean that up rather get the big one merged

**Comment by tausifmuzaffar** [comment]
@shriramshankar the tests are passing now

**Comment by tausifmuzaffar** [comment]
@iamigo @shriramshankar you both have approved this review right? on the top it says only Shriram has reviewed it
